heron/stmgr/src/cpp/manager/instance-server.cpp

```diff
@@ -356,6 +358,10 @@ void InstanceServer::DrainTupleStream(proto::stmgr::TupleStreamMessage* _message
   TaskIdInstanceDataMap::iterator iter = instance_info_.find(task_id);
   if (iter == instance_info_.end() || iter->second->conn_ == NULL) {
     LOG(ERROR) << "task_id " << task_id << " has not yet connected to us. Dropping...";
+  } else if (droptuples_upon_backpressure_ && iter->second->conn_->hasCausedBackPressure()) {
+    LOG(ERROR) << "task_id " << task_id << " is causing backpressure, so Dropping tuples to him";
+    // Ideally we would have counted the numbers lost, but we cannot since the tuples are in
```

Comment:
Do we have any information for the package we can log? Like time window, etc? Sorry if this is a silly question.

Reply:
Addressed by adding bytes dropped